tests/containers/integration-test-base

```diff
@@ -2,6 +2,8 @@ FROM quay.io/centos/centos:stream9
 
 RUN dnf install -y dnf-plugin-config-manager
 
+RUN dnf upgrade --refresh -y
```

Comment:
Hmm, we are already getting the latest CS9 container as a base here, so does it mean that the container might not be synced with the actual content of CS9 repos? Meaning that for example container is recreated only once per day while repos are updated regularly during the day?

Reply:
Since we don't have any control over the base image we cannot guaranty what's in it and what will be there the following day. As a result using `dnf upgrade` is the right way to go.
I would argue that this line could have been merged with the next one, making it a single layer. But, since it's a test image, it's not that important.